recipe_scrapers/plugins/exception_handling.py

```diff
@@ -1,12 +1,26 @@
 import functools
 import logging
 
+from recipe_scrapers.settings import settings
+
 from ._interface import PluginInterface
 
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
 
 class ExceptionHandlingPlugin(PluginInterface):
     """
-    TODO: write docstring
+    Plugin that is used only if settings.EXCEPTION_HANDLING is set to True.
```

Comment:
Nit: could we call this `settings.SUPPRESS_EXCEPTIONS`?

Reply:
I like `SUPPRESS_EXCEPTIONS` more too, adjusted

Reply:
Thanks!